client/state/action-types.js

```diff
@@ -61,4 +61,8 @@ export const SUPPORT_USER_TOKEN_FETCH = 'SUPPORT_USER_TOKEN_FETCH';
 export const SUPPORT_USER_TOKEN_SET = 'SUPPORT_USER_TOKEN_SET';
 export const SUPPORT_USER_RESTORE = 'SUPPORT_USER_RESTORE';
 export const USER_RECEIVE = 'USER_RECEIVE';
+export const VIDEO_FETCH = 'VIDEO_FETCH';
```

Comment:
Already some inconsistency with the action type naming in this file :frowning: See also: `_REQUEST`, `_REQUEST_SUCCESS`, `_REQUEST_FAILURE`. I'm partial to the latter, but we should really settle on a consistent approach. Thoughts?

For reference, the idea of using "request" is such that not all network requests are expecting data (even `GET` requests), so "fetch" is not always suitable.